mindmeld/cli.py

```diff
@@ -26,32 +26,26 @@
 import sys
 import time
 import warnings
-
 from shutil import which
+
 import click
 import click_log
 import distro
 import requests
 from tqdm import tqdm
 
+from . import auto_annotator  # noqa: F401 pylint: disable=W0611
```

Comment:
looks like this is an unused import

Reply:
This import is needed, since without it the Annotators would not register (bottom of auto_annotator.py) and would not be loaded dynamically for CLI commands

Reply:
Could the registration method not be called explicitly within the ``create_annotator`` method in ``helpers.py``. That way the implicit/unused import can be done away with here?

Reply:
These are good suggestions and either could work, however, there are some design related trade-offs to consider. (Note: The registration functions add to an ANNOTATOR_MAP = {} in models/helpers.py.)

We could call these functions in the `annotate` command in cli.py. However, if `create_annotator()` is used in a different file in the future, we'd have to register these classes again. Also, a user creating a custom annotator and using the cli will have to update both `cli.py` and `auto_annotator.py`, they will also need to update the imports at the top of `cli.py`

If we call it in `create_annotator()` itself in `helpers.py` each class would be registered every time the function is called. This wouldn't be consistent with how the other MAPs are used. Users would need also need to update `models/helpers.py` to use a custom annotator in cli.py.

Reply:
We could alternatively, wrap all of the register functions at the bottom of auto_annotator.py into a single function register_all_annotators(). Then import and call this function explicitly in cli.py. This should avoid the implicit/unused import issue, and keeps the registration of annotators in the annotator file itself.

```
def register_all_annotators():
    register_annotator("SpacyAnnotator", SpacyAnnotator)
    register_annotator("BootstrapAnnotator", BootstrapAnnotator)
    register_annotator("MultiLingualAnnotator", MultiLingualAnnotator)
```

Reply:
@KKsharma99 That makes sense.

Reply:
Random thought that is beyond the scope of this change: the MindMeld CLI takes a very long time to run -- even for a simple version check `mindmeld -V`. We can improve this by minimizing the number of top-level imports in the cli. Something to keep in mind :-)